mantle/platform/qemu.go

```diff
@@ -240,17 +238,22 @@ func (inst *QemuInstance) Destroy() {
 // Currently effective on aarch64: switches the boot order to boot from disk on reboot. For s390x and aarch64, bootindex
 // is used to boot from the network device (boot once is not supported). For s390x, the boot ordering was not a problem as it
 // would always read from disk first. For aarch64, the bootindex needs to be switched to boot from disk before a reboot
-func (inst *QemuInstance) SwitchBootOrder() error {
+func (inst *QemuInstance) SwitchBootOrder() (err error) {
 	if system.RpmArch() != "s390x" && system.RpmArch() != "aarch64" {
 		//Not applicable for other arches
 		return nil
 	}
 	monitor, err := newQMPMonitor(inst.tempdir)
 	if err != nil {
+		return errors.Wrapf(err, "Could not create QMP connection")
```

Comment:
`errors.Wrapf` is out of style....using `fmt.Errorf("Could not create QMP connection: %w")` is the preferred way.